### DISCUSSION TITLE
address not accepted for registration

### DISCUSSION THREAD

**Discussion by rat1402** [simple-login/app#1222] [Q&A]
Hi

I wonder whether others had a similar experience: I ran into problems with some online registration when they didn't allow an e-mail address from one or any of the domains offered by SimpleLogin for registration. The error message said that it is a "temporary e-mail" and only "real, permanent" addresses are accepted. In some instances I was able to circumvent the problem by creating an alias with another SimpleLogin domain instead of the one for which I have a subdomain. In other cases, none of the domains worked (but I created a lot of aliases for testing, which remain disabled).

I am afraid that this will become more common in the future when various providers become more and more aware of which domains are used here. But this defeats the purpose of having a flexible solution of not being tied to a given e-mail service for the "permanent address". *sigh*

Any similar experiences, any suggestions of how to solve this?

**Comment by nguyenkims** [comment]
@rat1402 Unfortunately some websites consider SimpleLogin aliases as temporary email addresses and block them. These websites often don't care about our privacy and maybe only accept popular email services like Gmail, Outlook. In this case we recommend to get in touch with the website to change their policy. If this isn't possible, the only solution is to use a secondary Gmail/outlook account.
